app/main.py

```diff
@@ -1,3 +1,64 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
```

Comment:
Try to separate this function into a few functions with different responsibilities.